cmd/template/dbdriver/files/service/postgres.tmpl

```diff
@@ -34,6 +34,7 @@ var (
 	username = os.Getenv("DB_USERNAME")
 	port     = os.Getenv("DB_PORT")
 	host     = os.Getenv("DB_HOST")
+  schema   = os.Getenv("DB_SCHEMA")
```

Comment:
fix the position

Reply:
Yup - @ashupednekar please fix the format and this LGTM